### ISSUE TITLE
Yes/No fields to boolean in Inspections dataset

### ISSUE THREAD

**Issue by mmartin78** [open-data-standards/permitdata.org#59]
Should Yes/No fields like Final and ReInspection be redefined to BOOL type instead of TEXT with Yes/No. This is usually more efficient and friendly to Database Systems and Programming Languages.

**Comment by eddietejeda** [comment]
To be as generic as possible, we're mapping to SQLITE datatypes. https://www.sqlite.org/datatype3.html. Actually, I noticed we might need to make a few more adjustments.

In this case, I agree. We should do away with Yes/No options and instead use 1/0.

**Comment by mmartin78** [comment]
I would disagree that SQLITE types are any more generic than any other database, SQL-92 would have been my choice, but that is besides the point.

For the purposes of this standard, I dont believe that the exact name of the data type is so important, more important are the values allowed for this fields. I do agree that 1/0 is definitely a much better option.

**Comment by bettin** [comment]
Is there a reason this wasn't a BOOL or INT in the first place? Changing it now would need to be done in a major version change since it breaks compatibility with v1.0
